sys/net/gnrc/application_layer/dhcpv6/client.c

```diff
@@ -89,6 +89,44 @@ void dhcpv6_client_conf_prefix(unsigned iface, const ipv6_addr_t *pfx,
     }
 }
 
+bool dhcpv6_client_check_ia_na(unsigned iface)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+
+    return netif->ipv6.aac_mode & GNRC_NETIF_AAC_DHCP;
+}
+
+int dhcpv6_client_add_addr(unsigned iface, ipv6_addr_t *addr)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+
+    DEBUG("DHCPv6 client: ADD IP ADDRESS\n");
+
+    return gnrc_netif_ipv6_addr_add(netif, addr, 64, 0);
+}
+
+void dhcpv6_client_deprecate_addr(unsigned iface, const ipv6_addr_t *addr)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+    int i;
```

Comment:
That was one whitespace removed to much ^^ (I know, now I am getting pedantic)

```suggestion
    int i;

```

(in the original suggestion I only wanted you to remove the four trailing whitespaces, but not the newline ;-))

Reply:
Oops. Sorry 😅